### DISCUSSION TITLE
Welcome to pickaxe Discussions!

### DISCUSSION THREAD

**Discussion by ulerdogan** [ulerdogan/pickaxe#25] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by richardsric7** [comment]
Are you accepting contributors to this project? I am interested.

**Reply by ulerdogan** [reply]
Hey, sure! 

I would love to see contributors. How'd you like to contribute?
